Spigot-Server-Patches/0408-Add-BellRingEvent.patch

```diff
@@ -0,0 +1,41 @@
+From dc507bbd88bc391eebff1a4c9ccf8c86bf466bfe Mon Sep 17 00:00:00 2001
+From: Eearslya Sleiarion <eearslya@gmail.com>
+Date: Mon, 24 Jun 2019 21:27:32 -0700
+Subject: [PATCH] Add BellRingEvent
+
+Add a new event, BellRingEvent, to trigger whenever a player rings a
+village bell. Passes along the bell block and the player who rang it.
+
+diff --git a/src/main/java/net/minecraft/server/BlockBell.java b/src/main/java/net/minecraft/server/BlockBell.java
+index e83108e3c..248bbb089 100644
+--- a/src/main/java/net/minecraft/server/BlockBell.java
++++ b/src/main/java/net/minecraft/server/BlockBell.java
+@@ -2,6 +2,12 @@ package net.minecraft.server;
+ 
+ import javax.annotation.Nullable;
+ 
++// Paper start
```

Comment:
Do not add imports, it adds maitenance burden

Reply:
That was actually not my import. That was there right after I added `BlockBell` to the import script and re-patched, so I assumed it was vanilla.

Reply:
he means the imports below on lines 18-20